scripts/test-turbo-modules.sh

```diff
@@ -175,6 +175,7 @@ create_library() {
     --languages cpp \
     --type module-new \
     --example $example_type \
+    --local false \
```

Comment:
This declines generating a local library when running the script from the root folder of another project (such as ubrn itself).

```
? Looks like you're under a project folder. Do you want to create a local library? › (Y/n)
```